programmers2/예산-수정.js

```diff
@@ -0,0 +1,12 @@
+function solution(d, budget) {
+    const price = d.sort((a,b) => a - b);
+    let cnt = 0;
+    
+        for(let i = 0; i < price.length; i++) {
+            if(budget >= price[i]) {
+                cnt ++;
+                budget -= price[i];
+            }
+        }
```

Comment:
들여쓰기가 안되어있다면 상사한테 뺨다구 7만대 기대하시면 됩니다 ㅋㅋㅋ
for와 if 뒤에는 공백 필수!
```suggestion
    for (let i = 0; i < price.length; i++) {
        if (budget >= price[i]) {
            cnt ++;
            budget -= price[i];
        }
    }
```